frontend/src/components/common/Resource/EditorDialog.tsx

```diff
@@ -49,12 +59,12 @@ import SimpleEditor from './SimpleEditor';
 type KubeObjectIsh = Partial<KubeObjectInterface>;
 
 export interface EditorDialogProps extends DialogProps {
-  /** The object to edit, or null to make the dialog be in "loading mode". Pass it an empty object if no contents are to be shown when the dialog is first open. */
-  item: KubeObjectIsh | null;
+  /** The object(s) to edit, or null to make the dialog be in "loading mode". Pass it an empty object if no contents are to be shown when the dialog is first open. */
+  item: KubeObjectIsh | object | object[] | string | null;
   /** Called when the dialog is closed. */
   onClose: () => void;
-  /** Called when the user clicks the save button. */
-  onSave: ((...args: any[]) => void) | null;
+  /** Called by a component for when the user clicks the save button. When set to "default", internal save logic is applied. */
+  onSave?: ((...args: any[]) => void) | string | null;
```

Comment:
minor nit: If the only string value this prop can accept then it should be represented in its' type
`((...args: any[]) => void) | "default" | null`